src/app/filter.pipe.ts

```diff
@@ -0,0 +1,33 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Todo } from './todo';
+import { ListComponent } from './list.component';
+
+@Pipe({
+  name: "sort"
+})
+export class FilterPipe implements PipeTransform {
+  transform(array: Array<any>, args: string): Array<string> {
+    {
+      array.sort((a: any, b: any) => {
```

Comment:
use a fine variable name, not just a/b/s/. change it

Reply:
ok sir!